src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -159,12 +159,9 @@
     <Compile Include="System\Drawing\Imaging\ImageCodecInfo.cs" />
     <Compile Include="System\Drawing\Imaging\ImageCodecInfoPrivate.cs" />
     <Compile Include="System\Drawing\Imaging\MetafileFrameUnit.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\User32\Interop.LOGFONT.cs"
-             Link="Common\Interop\Windows\User32\Interop.LOGFONT.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Gdi32\Interop.RasterOp.cs"
-             Link="Common\Interop\Windows\Gdi32\Interop.RasterOp.cs" />
-    <Compile Include="$(CommonPath)System\Text\ValueStringBuilder.cs"
-             Link="Common\System\Text\ValueStringBuilder.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\User32\Interop.LOGFONT.cs" Link="Common\Interop\Windows\User32\Interop.LOGFONT.cs" />
```

Comment:
I believe VS is changing the `Link` to the same line as the include, but if I remember correctly there was an effort in the repo to put it in a separate line because they could get large and wouldn't fir the screen

Reply:
Yeah, VS changed this. Rather avoid the busy work for future changes, but if this is some sort of policy we can get it better documented and change it back. :)